### DISCUSSION TITLE
Inline SVG doesn't display

### DISCUSSION THREAD

**Discussion by laike9m** [gonzalezreal/swift-markdown-ui#356] [Ideas]
**Describe the bug**
Inline SVG doesn't display

**Checklist**
- [x] I can reproduce this issue with a vanilla SwiftUI project.
- [x] I can reproduce this issue using the `main` branch of this package.
- [x] This bug hasn't been addressed in an [existing GitHub issue](https://github.com/gonzalezreal/swift-markdown-ui/issues).

**Steps to reproduce**
Explanation of how to reproduce the incorrect behavior. This could include an attached project, a link to code, or a Markdown-formatted text exhibiting the issue.

```
Markdown(
            """
            ![](https://dev.w3.org/SVG/tools/svgweb/samples/svg-files/atom.svg)
            """)
```

**Expected behavior**
Image shows

**Screenshots**
Nothing shows

**Version information**
- MarkdownUI: 2.3.1
- OS: macOS 14.7
- Xcode: 16.0

**Comment by gonzalezreal** [comment]
Hi,
MarkdownUI does not currently support SVG, nor will do in the near feature. I moved the issue into a discussion to keep track of it as a feature request.

**Reply by laike9m** [reply]
I see. Thanks for the quick response.
